services/frontend/compose.yaml

```diff
@@ -13,5 +13,6 @@ services:
       - /config/init.sh && nginx -g "daemon off;"
     environment:
       BE_VERSION: ${BE_VERSION:-v4}
+      LDAP_ENABLED: ${LDAP_ENABLED:-}
```

Comment:
Is the default needed?

Reply:
Not needed per say, but otherwise you would get a warning